script/repack-archive.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
+import optparse
+import sys
+import os
+import subprocess
+from lib.util import scoped_cwd, make_zip, tempdir
+
+
+def GetLZMAExec(build_dir):
+  if sys.platform == 'win32':
+    lzma_exec = os.path.join(build_dir, "..", "..", "third_party",
+                             "lzma_sdk", "Executable", "7za.exe")
+  else:
+    lzma_exec = '7zr'  # Use system 7zr.
+  return lzma_exec
```

Comment:
insconsistent idents with `main()`

Reply:
other pylint issues can be found here https://ci.brave.com/job/pr-brave-browser-fix-win-release-archive-noplatform/5/pylint/

Reply:
Fixed